.github/workflows/send-tag-event.yml

```
@@ -0,0 +1,24 @@
+name: Send Tag Event
+
+on:
+  push:
+    tags:
+      - '*'
```

```suggestion
      - '!*-integration'
```

If you only want to receive dev and latest tags.

---

if we don't see values in using the integration tags with accelerate, then yeah, let's remove them!